packages/react-ui-core/src/Gmap/utils/__tests__/mapEventHelpers-test.js

```diff
@@ -58,6 +58,32 @@ describe('mapEventHelpers', () => {
     expect(actual).toEqual(expected)
   })
 
+  it('sets up events with addListenerOnce if the once argument is set to true', () => {
```

Comment:
Looks like you rebases this whole section wrong: https://github.com/rentpath/react-ui/blob/master/packages/react-ui-core/src/Gmap/utils/__tests__/mapEventHelpers-test.js